src/style.css

```diff
@@ -0,0 +1,119 @@
+body,
+div,
+h2,
+p {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+}
+
+.card {
```

Comment:
checklist
[FILE STRUCTURE] - You should create separate file for each BEM block